messages/scratchOrgCreate.md

```diff
@@ -25,3 +25,19 @@ Expected 'retry' to be an integer number.
 # WaitValidationMaxError
 
 Expected 'wait' greater than or equal to %s but received %s.
+
+# NoScratchOrgInfoError
+
+No ScratchOrgInfo was found. Check that the ID and the DevHub are correct
+
+# ScratchOrgDeletedError
+
+That scratch org has been deleted and you can't connect to it
+
+# StillInProgressError
+
+The scratch org is not ready yet (Status = %)
+
+# action.StillInProgress
+
+Wait and try the <%= config.bin %> <%= command.id %> command again
```

Comment:
```suggestion
Wait and try the <%= config.bin %> <%= command.id %> command again. 
```

Reply:
Question: Can we add a bit more info about how long to wait?  Yeah, I know, it's impossible to know.  But it would be good to give some idea, such as wait for a few minutes or hours or days, you know?  For example "Wait for a few minutes, and then try the ....".